modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/pojo/index/ExperimentTagsByApplication.java

```diff
@@ -37,10 +38,14 @@
     @Column(name = "app_name")
     String appName;
 
+    @PartitionKey(1)
```

Comment:
I checked in other places in the code. You might want to probably declare it as a clustering column instead.